DESCRIPTION

```diff
@@ -11,17 +11,14 @@ Description: A programmatic interface to the Web Service methods
 Version: 1.1.0.9113
 License: MIT + file LICENSE
 Authors@R: c(
-    person("Scott", "Chamberlain", role = c("aut", "cre"),
-      email = "myrmecocystus@gmail.com",
-      comment = c(ORCID="0000-0003-1444-9135")),
-    person("Vijay", "Barve", role = "ctb"),
-    person("Dan", "Mcglinn", role = "ctb"),
-    person("Damiano", "Oldoni", role = "ctb"),
+    person("Scott", "Chamberlain", role = c("aut", "cre"), email = "myrmecocystus@gmail.com", comment = c(ORCID="0000-0003-1444-9135")),
+    person("Vijay", "Barve", role = "ctb", comment = c(ORCID = "0000-0002-4852-2567")),
+    person("Dan", "Mcglinn", role = "ctb", comment = c(ORCID = "0000-0003-2359-3526")),
+    person("Damiano", "Oldoni", role = "ctb", comment = c(ORCID = "0000-0003-3445-7562")),
     person("Laurens", "Geffert", role = "ctb"),
-    person("Karthik", "Ram", role = "ctb")
+    person("Karthik", "Ram", role = "ctb", comment = c(ORCID = "0000-0002-0233-1757"))
     )
-URL: https://github.com/ropensci/rgbif (devel) 
-    https://ropensci.github.io/occurrence-manual/ (manual)
+URL: https://github.com/ropensci/rgbif, https://ropensci.github.io/rgbif
```

Comment:
can we do all 3 urls? that is, something like: 

`https://github.com/ropensci/rgbif (devel)  https://ropensci.github.io/rgbif (docs)  https://ropensci.github.io/occurrence-manual/ (manual)`

Reply:
3 URLs are fine I think, but they should be comma separated (see http://r-pkgs.had.co.nz/description.html):

> Multiple URLs are separated with a comma.

Not sure about adding `(devel)` etc. Would keep it to URL only.

Reply:
sounds good on the comma